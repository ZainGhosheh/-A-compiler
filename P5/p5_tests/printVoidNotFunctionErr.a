a : () -> int {
	return 1;
}

z : () -> void {
}

f : () -> void {
	toconsole a();
	toconsole z(); # error
}
