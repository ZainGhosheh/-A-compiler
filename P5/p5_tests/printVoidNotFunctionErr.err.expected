FATAL [10,2]-[10,15]: Attempt to output void
Type Analysis Failed
